### DISCUSSION TITLE
Please separate the unused variable checker from the auto-arrange code

### DISCUSSION THREAD

**Discussion by barbarossa69** [ScratchAddons/ScratchAddons#3254] [Ideas]
Hi, thanks to everyone for this addon, but I have a request.

I really like the unused variable checking part of the auto-arrange code, but I don't like the auto-arrange code itself - I prefer the vertical approach by the native scratch editor, so would it be possible to separate the unused variable checking part out from this, so it's got its own option on the right-click menu?

Thanks for reading!

**Comment by CST1229** [comment]
Maybe there could be some checkboxes to separate the different features of the improved cleanup: 2,0-style positioning, removing unused variables and removing orphaned reporters.

**Comment by WorldLanguages** [comment]
At some point we'll split editor-devtools into multiple addons, at that point we can probably make this more customizable.
